src/ResourceManager/IotCentral/Commands.IotCentral/AzureRM.IotCentral.psd1

```diff
@@ -0,0 +1,132 @@
+#
+# Module manifest for module 'AzureRM.IotCentral'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 10/17/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'df9fc69a-c019-403a-9013-eefa7eb5e27a'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - IotCentral service cmdlets for Azure Resource Manager'
+
+# Minimum version of the Windows PowerShell engine required by this module
+# PowerShellVersion = '5.0'
+
+# Name of the Windows PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the Windows PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.5.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = '4.0'
```

Comment:
@cle5eland I would uncomment this field as well

Reply:
Done